cvmfs/kvstore.cc

```diff
@@ -0,0 +1,233 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include "kvstore.h"
+
+#include <assert.h>
+#include <errno.h>
+#include <limits.h>
+#include <string.h>
+
+#include <algorithm>
+
+#include "logging.h"
+#include "util_concurrency.h"
+
+using namespace std;  // NOLINT
+
+namespace kvstore {
```

Comment:
I think we should try to reduce the number of namespaces.  If it's not too big of a change, I think we should get rid of `namespace kvstore {`